modules/network-observability-resources-table.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resources-table{context}"]
+= Resource considerations
+The following table outlines examples of resource considerations for clusters with certain workload sizes. 
+
+[IMPORTANT]
+====
+The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Resource recommendations
+[options="header"]
+|============================================================================================================================
+|                                     | Extra small (10 Nodes) | Small (25 nodes)  | Medium (65 Nodes) ^[2]^ | Large (120 nodes) ^[2]^
+| *Worker Node vCPU and Memory*       | 4 vCPUs\| 16GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem  ^[1]^  |16 vCPUs\| 64GiB Mem ^[1]^ 
+| *LokiStack Size*                    | 1x.extra-small         | 1x.small          | 1x.small           | 1x.medium
+| *NetObserv Controller Memory Limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi                
+| *eBPF Sampling Rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50 (default)
+| *eBPF Memory Limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default) 
+| *FLP Mem Limit*                     | 800Mi (default)        | 800Mi (default)   | 800Mi (default)    | 800Mi (default)         
+| *FLP Kafka Partitions*              | N/A                    | 48                | 48                 | 48            
+| *Kafka Consumer Replicas*           | N/A                    | 24                | 24                 | 24
+| *Kafka Brokers*                     | N/A                    | 3 (default)       | 3 (default)        | 3 (default)
+|============================================================================================================================
+1. Tested with AWS M6i instances.
+2. In addition to this worker and its controller, 3 infra nodes (size `M6i.12xlarge`) and 1 workload node (size `M6i.8xlarge`) were tested. 
```

Comment:
Suggestion: When using the `^[1]^` style of table-footnotes mentioned in the repo guidelines ([Alternative footnote styling in tables](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables)), I'd suggest using the `[.small]` style shown there, because it makes it show up more like what footnotes normally look like:

```suggestion
[.small]
--
1. Tested with AWS M6i instances.
2. In addition to this worker and its controller, 3 infra nodes (size `M6i.12xlarge`) and 1 workload node (size `M6i.8xlarge`) were tested. 
--
```

Reply:
Niiice, thanks for this suggestion :) Addressed.